examples/cli/src/test/java/io/unitycatalog/cli/access/CliAccessControlBaseCrudTest.java

```diff
@@ -0,0 +1,197 @@
+              add(
+                      Step.CommandStep.of(
+                              SUCCEED,
+                              "user",
+                              "create",
+                              "--name",
+                              "Principal 2",
+                              "--email",
+                              "principal-2@localhost"));
+              add(
+                      Step.CommandStep.of(
+                              SUCCEED,
+                              "user",
+                              "create",
+                              "--name",
+                              "Regular 1",
+                              "--email",
+                              "regular-1@localhost"));
+              add(
+                      Step.CommandStep.of(
+                              SUCCEED,
+                              "user",
+                              "create",
+                              "--name",
+                              "Regular 2",
+                              "--email",
+                              "regular-2@localhost"));
+            }
+          };
+
+  protected List<Step> commonSecurableSteps =
+          new ArrayList<>() {
+            {
+              // give user CREATE CATALOG
+              add(Step.TokenStep.of(SUCCEED, "admin"));
+              add(
+                      Step.CommandStep.of(
+                              SUCCEED,
+                              1,
+                              "permission",
+                              "create",
+                              "--securable_type",
+                              "metastore",
+                              "--name",
+                              "metastore",
+                              "--principal",
+                              "principal-1@localhost",
+                              "--privilege",
+                              "CREATE CATALOG"));
+
+              // create a catalog -> CREATE CATALOG -> allowed
+              add(Step.TokenStep.of(SUCCEED, "principal-1@localhost"));
+              add(
+                      Step.CommandStep.of(
+                              SUCCEED,
+                              "catalog",
+                              "create",
+                              "--name",
+                              "cat_pr1",
+                              "--comment",
+                              "(created from scratch)"));
+
+              // give user CREATE SCHEMA on cat_pr1
+              add(
+                      Step.CommandStep.of(
+                              SUCCEED,
+                              1,
+                              "permission",
+                              "create",
+                              "--securable_type",
+                              "catalog",
+                              "--name",
+                              "cat_pr1",
+                              "--principal",
+                              "principal-1@localhost",
```

Comment:
Should this be `principal-2@localhost` since the owner of `cat_pr1` is `principal-1@localhost`?